Author: Jo-Philipp Wich <jow@openwrt.org>
Date:   Sat Apr 21 03:02:39 2012 +0000

    gcc: add patch to make the getenv() spec function nonfatal if requested environment variable is unset
    
    SVN-Revision: 31390

---
 gcc/gcc.c |    6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

--- a/gcc/gcc.c
+++ b/gcc/gcc.c
@@ -9281,8 +9281,10 @@
     value = varname;
 
   if (!value)
-    fatal_error (input_location,
-		 "environment variable %qs not defined", varname);
+    {
+      warning (input_location, "environment variable %qs not defined", varname);
+      value = "";
+    }
 
   /* We have to escape every character of the environment variable so
      they are not interpreted as active spec characters.  A
